src/store/index.js

```diff
@@ -26,6 +27,9 @@ export default createStore({
       state.lang = language;
       i18n.global.locale = language;
     },
+    setOpenAIApiKey(state, openaiApiKey) {
```

Comment:
函数和变量名的驼峰命名，应该遵循统一原则。建议这样：

```suggestion
    setOpenaiApiKey(state, openaiApiKey) {
```